Carla-FollowLaneTraffic/tensorflow/PilotNet/utils/pilotnet.py

```diff
@@ -0,0 +1,31 @@
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.layers import Flatten, Dense, Conv2D, BatchNormalization, Dropout, ConvLSTM2D, Reshape, Activation, MaxPooling2D, InputLayer, LayerNormalization, Lambda
+from tensorflow.keras.optimizers import Adam
+from tensorflow.keras.layers import LeakyReLU
```

Comment:
The following imports are not needed:

from tensorflow.keras.layers import LeakyReLU
import tensorflow as tf